cubic-server/protocol/ClientPackets.cpp

```diff
@@ -79,8 +79,10 @@ std::shared_ptr<std::vector<uint8_t>> protocol::createSpawnPlayer(const SpawnPla
 std::shared_ptr<std::vector<uint8_t>> protocol::createEntityAnimation(EntityAnimationID animId, int32_t entityID)
 {
     std::vector<uint8_t> payload;
-    serialize(payload, entityID, addVarInt,
-        animId, addByte);
+    serialize(payload,
+        entityID, addVarInt,
+        (uint8_t)animId, addByte
```

Comment:
Same `uint8_t` 🤡

Reply:
But I can't remove the cast. If I do so, because animId is a EntityAnimationID, and not a uint8_t it will not compile

Reply:
We have the same problem everywhere else

Reply:
Oh yeah, enum class things 😅